projects/common/src/utilities/formatters/string/string-formatter.test.ts

```diff
@@ -26,4 +31,12 @@ describe('String formatter', () => {
     expect(displayString(Symbol('test symbol'))).toBe('Symbol(test symbol)');
     expect(displayString(false)).toBe('false');
   });
+
+  test('creates string array correctly from comma separated list', () => {
+    expect(getStringsFromCommaSeparatedList('first,second,   third   ')).toEqual(
+      expect.arrayContaining(['first', 'second', 'third'])
```

Comment:
use a regular equality match in these tests, since we're looking for this to return a specific array (i.e. it would be wrong if it returned `['first', 'second', 'third', 'fourth']` but this matcher would accept it.
```suggestion
      ['first', 'second', 'third']
```

Reply:
done